packages/reports/addon/components/grouped-list.hbs

```diff
@@ -7,22 +7,28 @@
     @bufferSize={{10}}
     @containerSelector="#{{this.guid}}"
     as |item idx|>
-    {{#if item.isGroup}}
-      <li
-        class="grouped-list__group-header"
-        role="button"
-        {{on "click" (fn this.toggleOpen item.name)}}
-        >
-        <DenaliIcon
-          class="grouped-list__group-header-icon grouped-list__group-header-icon--{{if item.isOpen "open" "closed"}}"
-          @icon="arrowhead-{{if item.isOpen "down" "right"}}"
-        />
-        <span class="grouped-list__group-header-content">{{item.name}} ({{item.length}})</span>
-      </li>
-    {{else}}
-      <li class="grouped-list__item m-l-18 m-t-2">
+    {{#if this.isSingleCategory}}
```

Comment:
Do we need this `if` can we use lines 28-30 to cover this case with a check on `isSingleCategory` for the indent class?